libs/community/langchain_community/document_loaders/github.py

```diff
@@ -127,32 +127,6 @@ def lazy_load(self) -> Iterator[Document]:
             else:
                 url = None
 
-    def load(self) -> List[Document]:
```

Comment:
a few of these, like this one, are nice for the API reference because they overwrite the docstring

Reply:
not super worried about this since lazy_load has the documentation